.github/workflows/ci_push.yml

```diff
@@ -20,9 +22,27 @@ jobs:
       run: echo "${{ secrets.GITHUB_TOKEN }}" | docker login ghcr.io -u ${{ github.repository_owner }} --password-stdin
 
     - name: Build, test, push docker
-      run: ./scripts/docker/build.sh --xenial --remote
-                                     --owner ${{ github.repository_owner }}
-                                     astrobee test_astrobee push_astrobee
+      run: >
+        revision="latest";
+        build_arg="astrobee";
+        push_arg="";
+        branch=${GITHUB_HEAD_REF:-${GITHUB_REF#refs/heads/}};
+        if [ $branch == "nasa" ]; then
```

Comment:
Branch nasa? This job is different from the others, was that intentional?

Reply:
this is a bug hahah thanks for spotting it!